src/els_config.erl

```diff
@@ -21,7 +21,6 @@
 %%==============================================================================
 %% Macros
 %%==============================================================================
--define(DEFAULT_CONFIG_PATH, "erlang_ls.config").
```

Comment:
After this change the code is still using this constant, so it would be nice to keep it.